functions/validator/utils.py

```diff
@@ -0,0 +1,30 @@
+__version__ = '0.0.1'
+
+import os
+import logging
+import time
+from datetime import datetime
+
+
+def set_stream_logger(name="cis-validator", level=logging.INFO,
+                      format_string=None):
+    """
+    """
+
+    if format_string is None:
```

Comment:
This is meant to only match on noneType to allow other classes to override the default logger formatting.  Otherwise if not set it goes to the default set of formatting.